src/components/common/buttons/SaveButton.js

```diff
@@ -43,7 +43,7 @@ const SaveButton = ({ children, onPress, onPressDone, doneDelay, styles, theme,
   return (
     <View style={styles.wrapper}>
       {state === DONE ? (
-        <TouchableOpacity cursor="inherit" style={[styles.iconButton, { backgroundColor }]}>
+        <TouchableOpacity cursor="inherit" style={[styles.iconButton, { backgroundColor }]} testId="save_button">
```

Comment:
should be `testID`

see src/components/appNavigation/TabsView.js as example

Reply:
okey